medusa/server/api/v2/search.py

```diff
@@ -0,0 +1,239 @@
+# coding=utf-8
+"""Request handler for statistics."""
+from __future__ import unicode_literals
+
+from collections import defaultdict
+from medusa import app
+from medusa.search.manual import collect_episodes_from_search_thread
+from medusa.tv.episode import Episode, EpisodeNumber
+from medusa.tv.series import Series, SeriesIdentifier
+from medusa.search.queue import (
+    BacklogQueueItem,
+    FailedQueueItem,
+    # ManualSearchQueueItem,
+    ForcedSearchQueueItem,
+)
+from medusa.server.api.v2.base import BaseRequestHandler
+
+from tornado.escape import json_decode
+
+
+class SearchHandler(BaseRequestHandler):
+    """Search queue request handler."""
+
+    #: resource name
+    name = 'search'
+    #: identifier
+    identifier = ('identifier', r'\w+')
+    #: path param
+    path_param = ('path_param', r'\w+')
+    #: allowed HTTP methods
+    allowed_methods = ('GET', 'POST')
+
+    def get(self, identifier, path_param=None, *args, **kwargs):
+        """Collect ran, running and queued searches for a specific show.
+
+        :param identifier:
+        :param path_param:
+        :type path_param: str
+        """
+        if not identifier:
+            return self._bad_request('You need to add the show slug to the route')
+
+        series = SeriesIdentifier.from_slug(identifier)
+        if not series:
+            return self._bad_request('Invalid series slug')
+
+        series_obj = Series.find_by_identifier(series)
+        if not series_obj:
+            return self._not_found('Series not found')
+
+        return {
+            'results': collect_episodes_from_search_thread(series_obj)
+        }
+
+    def post(self, identifier, path_param=None):
+        """Queue a backlog search for a range of episodes.
+
+        :param identifier:
+        :param path_param:
+        :type path_param: str
+        "tvdb1234" : [
+            "s01e01",
+            "s01e02",
+            "s03e03",
+        ]
```

Comment:
Was this part of the docstring added intentionally?

Reply:
yes. I've cleaned it up a little by adding it with the :example: section.